librz/bin/format/pe/pe.c

```diff
@@ -1443,16 +1443,11 @@ static int bin_pe_init_imports(struct PE_(rz_bin_pe_obj_t) * bin) {
 	int indx = 0;
 	int rr, count = 0;
 	int import_dir_size = data_dir_import->Size;
-	int delay_import_dir_size = data_dir_delay_import->Size;
 	/// HACK to modify import size because of begin 0.. this may report wrong info con corkami tests
 	if (!import_dir_size) {
 		// asume 1 entry for each
 		import_dir_size = data_dir_import->Size = 0xffff;
 	}
-	if (!delay_import_dir_size) {
```

Comment:
I think this shouuld not completely be removed but it should become something like:
```C
if (!data_dir_delay_import->Size) {
  data_dir_delay_import->Size = 0xffff;
}
```
Unless also `data_dir_delay_import->Size` is dead...